docker/docker-compose.yaml

```diff
@@ -254,6 +254,8 @@ services:
     volumes:
       # Mount the storage directory to the container, for storing user files.
       - ./volumes/app/storage:/app/api/storage
+    healthcheck:
+      test: ['CMD', 'curl', '-f', 'http://localhost:5001/health']
```

Comment:
This doesn't seems correct to me, I think this should not be hard coded.

Reply:
Could you show me the log after execute `docker inspect docker-api-1` ? I have already tested it locally

Reply:
His worry comes with the hardcoded value for binded IP and port , rather than use the system environment variables. Try to use `DIFY_BIND_ADDRESS` and `DIFY_PORT` like the way in `entrypoint.sh` entry script.

Reply:
I think using localhost does not mean hardcoding it. Generally, web services are designed to support access from the local machine. If need to change the hard-coded ip and port, I suggest adding a health check shell script in the Dockerfile. This script would reference the DIFY_BIND_ADDRESS  and DIFY_PORT environment variable, and then the healthcheck in docker-compose would execute this script to assess the service's health. However, I feel that this approach might be overcomplicating a relatively simple issue.

Reply:
Alignment is the key design and the prerequisite. There's no point in assuming it must be bonded onto localhost:5001.